docs/docs/coverage/iac/cloudformation.md

```diff
@@ -2,23 +2,31 @@
 Trivy supports the scanners listed in the table below.
 
 |      Scanner       | Supported |
-| :----------------: | :-------: |
+|:------------------:|:---------:|
 | [Misconfiguration] |     ✓     |
 |      [Secret]      |     ✓     |
 
 It supports the following formats.
 
 | Format | Supported |
-| :----: | :-------: |
+|:------:|:---------:|
 |  JSON  |     ✓     |
 |  YAML  |     ✓     |
 
 ## Misconfiguration
 Trivy recursively searches directories and scans all found CloudFormation files.
 It evaluates properties, functions, and other elements within CloudFormation files to detect misconfigurations.
 
+### Value Overrides
```

Comment:
How about adding examples of supported formats or a link to aws-cli v2?

Reply:
It's here https://github.com/aquasecurity/trivy/pull/5507/files#diff-e6c409fcf1992f2a1c44904cc9967f4d464c6fb3f8b30567d70f616812a6248dR32 which is referenced here https://github.com/aquasecurity/trivy/pull/5507/files#diff-e6c409fcf1992f2a1c44904cc9967f4d464c6fb3f8b30567d70f616812a6248dR21

Reply:
[This](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html) documentation describes the structure of template parameters, but not the format of parameters defined in the file.

Reply:
Added https://github.com/aquasecurity/trivy/pull/5507/commits/8d27d95d16b13c746b61ced87100d093f9f40744